lib/LaTeXML/Package/llncs.cls.ltxml

```diff
@@ -241,6 +241,9 @@ DefMacro('\homedir', "\~{ }");
 DefMacro('\idxquad', '\hskip 10pt\relax');
 #======================================================================
 
+# Newer llncs versions have an ORCID id macro
+DefConstructor('\orcidID{}', '<ltx:sup><ltx:ref href="https://orcid.org/#1">#1</ltx:ref></ltx:sup>');
```

Comment:
Seems like this should be frontmatter/metadata?

Reply:
indeed, it is used as a superscript on the author's name in the main frontmatter author list. It can be enhanced, just not sure what the perfect tag setup would be.